.github/workflows/sarplus.yml

```
@@ -36,10 +36,10 @@ jobs:
     # Test pysarplus with different versions of Python.
     # Package pysarplus and upload as GitHub workflow artifact when merged into
     # the main branch.
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

We used to have ubuntu-latest, but then we fixed it because GitHub changed the env and many of our tests broke